packages/rancher-vsphere/rancher-vsphere-csi/package.yaml

```
@@ -1,4 +1,4 @@
 url: https://github.com/rancher/vsphere-charts.git
 subdirectory: charts/rancher-vsphere-csi
-commit: 96687762c6642efc3650626a0738c8caa45ae7c3
-version: 102.0.0
+commit: 8b8e8cf13e9c971330bf96517fab1deec1f23b05
+version: 103.0.0
```

We should bump the minor version in the event of adding support for new k8s versions. 
```suggestion
version: 102.1.0
```